src/runOnPullRequest.js

```diff
@@ -22,6 +22,8 @@ import {
     MATCH_COMMENT_HEADER_REGEX,
 } from './constants';
 
+type NameToLabelToFiles = {[name: string]: {[label: string]: string[], ...}, ...};
```

Comment:
Before, if there were multiple matching rules, I think it would do:
```
{
  lilli: [fileFromR1, fileFromR2, fileFromR2]
...
}
```

Then in the comment it would do:
```
lilli because fileFromR1, fileFromR2, fileFromR2
```

But now if there are multiple matching rules, I think I would want:
```
lilli (rule1): fileFromR1
lilli (rule2): fileFromR2, fileFromR2
```

I'm not clear if this structure gets us to that.

Reply:
Yes, it does!  You can see in some of the test output.

Reply:
Maybe it does, but the syntax is starting to twist my head...

Reply:
If js allowed non-string map keys, I could do a map from `(person, label)` to `[files]`, but it doesn't, so this is the best I could do...